realtime/utils.js

```diff
@@ -211,7 +211,7 @@ function perspectiveEmit(nspComponents, obj) {
   }
 
   /*
-   * if this is a subject object, just apply the subjcTagFilter and return
+   * if this is a subject object, just apply the subjecTagFilter and return
```

Comment:
still missing a letter here, obviously not blocking, still waiting for @shriramshankar review

Reply:
lol sorry brain fart